OWNERS_ALIASES

```diff
@@ -14,50 +14,57 @@ aliases:
     - ixdy
     - mikedanese
     - tpepper
+  release-team:
+    - aishsundar
+    - jberkus
+    - justaugustus
+    - spiffxp
+    - tpepper
   release-team-lead-role:
-    - jberkus # 1.11
     - tpepper # 1.12
     - aishsundar # 1.13
     - spiffxp # 1.14
+    - claurence # 1.15
   patch-release-manager-role:
     - foxish # 1.11
     - feiskyer # 1.12
     - aleksandra-malinowska # 1.13
     - tpepper # 1.13
   ci-signal-role:
-    - aishsundar # 1.11
     - mohammedzee1000 # 1.12
     - jberkus # 1.13
     - mariantalla # 1.14
+    - alejandrox1 # 1.15
   enhancements-role:
-    - idvoretskyi # 1.11
     - justaugustus # 1.12
-    - kacole2 # 1.13
+    - kacole2 # 1.15 / 1.13
     - claurence # 1.14
   test-infra-role:
-    - bentheelder # 1.11
     - cjwagner # 1.13 / 1.12
     - amwat # 1.14
+    - imkin # 1.15
   bug-triage-role:
-    - tpepper # 1.11
     - guineveresaenger # 1.12
-    - nikopen # 1.13 / 1.14
+    - nikopen # 1.14 / 1.13
+    - soggiest # 1.15
   branch-manager-role:
-    - calebamiles # 1.11
     - dougm # 1.13 / 1.12
     - hoegaarden # 1.14
+    - bubblemelon # 1.15
   docs-role:
-    - misty # 1.11
     - zparnold # 1.12
     - tfogo # 1.13
     - jimangel # 1.14
+    - MAKOSCAFEE # 1.15
   release-notes-role:
     - nickchase # 1.12 / 1.11
     - marpaia # 1.13
-    - dstrebel # 1.10
+    - dstrebel # 1.14
+    - onyiny-ang # 1.15
   communications-role:
     - kbarnard10 # 1.13 / 1.12 / 1.11
     - nwoods3 # 1.14
+    - castrojo # 1.15
   product-security-committee:
```

Comment:
PSC member has gotten stale versus https://github.com/kubernetes/security/blob/master/security-release-process.md#product-security-committee-psc

Reply:
@tpepper -- thanks for the catch. I've updated the PSC alias!